app/views/katello/api/v2/content_view_filter_rules/matching_content.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @rule
+
+attributes :id
+
+node(:matching_content) do |_|
+  @rule.class.name == Katello::ContentViewPackageFilterRule.to_s ? @rule.matching_rpms : []
```

Comment:
This will error out  in the controller if an unsupported filter rule type is used, so a blank array should never be shown, but I added it in case anything slips by

Reply:
Should we show an error if there is an unsupported filter rule type?

Reply:
An API call with `matching_content` to any rule that isn't a package filter rule will return an error saying this rule type isn't supported yet, I just added a way to catch this in the rabl in case anything gets by. The UI won't show any buttons for non-package rules so the user can't calll for matching content there.

The implementation has changed with the API change, but I kept the same error when it's not a package rule.